docs/content/latest/benchmark/performance-troubleshooting.md

```diff
@@ -0,0 +1,150 @@
+Debug logs are output to `/home/yugabyte/{master|tserver}/logs`.
+
+### View Standard Output/Standard Error (stderr/stdout) of yb-tserver & yb-master process
+
+These go to e.g. `/home/yugabyte/tserver/tserver.{err|out}.
+
+## Setting gflags dynamically
+
+The `yb-ts-cli` utility allows changing gflags dynamically. _Note: For this step you must identify the server using its RPC port (not the HTTP port).
+
+For example, to increase the verbose logging level to 2:
+
+    $ ./yb-ts-cli --server_address=localhost:9100 set_flag v 2
+
+Or, to turn tracing on for all RPCs:
+
+    $ ./yb-ts-cli --server_address=localhost:9100 set_flag rpc_dump_all_traces 1
+
+
+## Slow Response Logs
+
+Slow responses (which take more than 75% of the configured RPC timeout) are already logged at WARNING level in this format, with a breakdown of time in various stages.
+
+```
+W0325 06:47:13.032176 116514816 inbound_call.cc:204] Call yb.consensus.ConsensusService.UpdateConsensus from 127.0.0.1:61050 (request call id 22856) took 2644ms (client timeout 1000).
+W0325 06:47:13.033341 116514816 inbound_call.cc:208] Trace:
+0325 06:47:10.388015 (+     0us) service_pool.cc:109] Inserting onto call queue
+0325 06:47:10.394859 (+  6844us) service_pool.cc:170] Handling call
+0325 06:47:10.450697 (+ 55838us) raft_consensus.cc:1026] Updating replica for 0 ops
+0325 06:47:13.032064 (+2581367us) raft_consensus.cc:1242] Filling consensus response to leader.
+0325 06:47:13.032106 (+    42us) spinlock_profiling.cc:233] Waited 2.58 s on lock 0x108dc3d40. stack: 0000000103d63b0c 0000000103d639fc 00000001040ac908 0000000102f698ec 0000000102f698a4 0000000102f93039 0000000102f7e124 0000000102fdcf7b 0000000102fd90c9 00000001\
+02504396 00000001032959f5 0000000103470473 0000000103473491 00000001034733ef 000000010347338b 000000010347314c
+0325 06:47:13.032168 (+    62us) inbound_call.cc:125] Queueing success response
+```
+
+## Turning RPC tracing on
+
+To turn tracing on for all RPCs, not just the slow ones, use the `rpc_dump_all_traces` gflag.
+
+## Print contents of a proto file
+
+To dump the contents of a file containing a proto (such as a file in the consensus-meta or tablet-meta directory), use 'yb-pbc-dump' utility:
+```
+$ ./yb-pbc-dump /mnt/d0/yb-data/tserver/consensus-meta/dd57975ef2f2440497b5d96fc32146d3
+$ ./yb-pbc-dump /mnt/d0/yb-data/tserver/tablet-meta/bfb3f18736514eeb841b0307a066e66c
+```
+
+Note: On mac, the environment variable DYLD_FALLBACK_LIBRARY_PATH needs to be set for pbc-dump to work. Add the following to ~/.bash_profile
+```
+export DYLD_FALLBACK_LIBRARY_PATH=~/code/yugabyte/build/latest/rocksdb-build
+```
+
+## yb-ts-cli 
+You can run various tablet related commands with yb-ts-cli by pointing at the master:
+```
+./yb-ts-cli list_tablets --server_address=localhost:8101
+./yb-ts-cli dump_tablet --server_address=localhost:8101 e1bc59288ee849ab850ae0a40bd88649
+```
+
+## yb-admin
+You can run various commands with `yb-admin`:
```

Comment:
I'd probably change the master addresses here to `127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100` and make a note in advance that this should point to the full set of master ip:ports